src/condor_contrib/docker/README.md

```diff
@@ -140,3 +140,6 @@ Known Issues
   `/update-config` in the container.  After updating secrets, run
   `/update-secrets` in the container.
 
+- cgroups support is not yet implemented.
+
+- Docker universe support is not yet implemented.
```

Comment:
Other things that probably don't work: Singularity

Reply:
I think if I installed Singularity into the image, it might work. Some of the worker nodes made by OSG folks have Singularity installed in them but I don't know if they're run privileged or not.